src/main/java/org/asteriskjava/manager/event/QueueMemberEvent.java

```diff
@@ -396,4 +397,8 @@ public Integer getWrapuptime() {
     public void setWrapuptime(Integer wrapuptime) {
         this.wrapuptime = wrapuptime;
     }
+
+    public Integer getLogintime() { return logintime; }
+
+    public void setLogintime(Integer logintime) { this.logintime = logintime; }
```

Comment:
Please fix formatting (not in the one line).

Reply:
I can change it if you want, but I followed the last formatting from you and other user as shown in the image below

<img width="748" alt="Screenshot 2023-04-19 at 15 03 12" src="https://user-images.githubusercontent.com/984668/233161539-3f72588f-af8f-41a9-8d24-85c1b79da4f5.png">